source/dataplaneapi/app.py

```diff
@@ -327,13 +327,15 @@ def create_asset():
         }
 
     Returns:
-        A dict containing the asset id and the location of the media object in the dataplane.
+        A dict containing the asset id and its designated location for metadata in the dataplane.
          .. code-block:: python
 
             {
                 "AssetId": asset_id,
-                "S3Bucket": dataplane_s3_bucket,
-                "S3Key": key
+                "AssetMetadataBucket": dataplane_s3_bucket,
+                "AssetMetadataFolder": dataplane_s3_key,
+                "S3Bucket": source_bucket,
+                "S3Key": source_key
```

Comment:
Prior to this PR, S3Bucket and S3Key used to point to the media file that was copied by the dataplane api into the /private/assets/ folder. Now that the dataplane api doesn't copy that file anymore, I've changed S3Bucket and S3Key to point to the user-specified source of the input media file. I added AssetMetadataBucket and AssetMetadataFolder because some operators used the old S3Bucket and S3Key to figure out where to put output metadata. Now they can use those two new keys instead.